handler.py

```diff
@@ -107,12 +130,30 @@ def getProject(project_name, created_at):
     else:
         return "Project not found."
 
+
 #=========================================#
 #=======       API Endpoints      ========#
 #=========================================#
 
 def addNewEvent(event, context):
-    
+    """
+    Endpoint to add a new event (reservation) to the calendar.
+
+    Parameters:
+        event.body.event_id (str): 
+            Unique id generated for each new event (eg. '999xx09b-xxxx-...')
+        event.body.start (str): 
+            UTC datestring of starting time (eg. '2022-05-14T17:30:00Z')
+        event.body.site (str): sitecode (eg. 'saf')
+
+    Returns:
+        POST the JSON body of project details with 
+        status code 200 to the calendar database, if successful
+
+    Raises:
+        Exception if request is not successful
```

Comment:
This doesn't actually raise an exception, the exception is caught in the catch block and the handler just returns a 200 response with the error details. Not actually a great way to handle errors; we'd be better off returning a 4xx or something since 200 implies success.